MANIFEST.in

```diff
@@ -4,3 +4,4 @@ include dev-requirements.txt
 include versioneer.py
 include src/fidesctl/api/ctl/alembic.ini
 include src/fidesctl/templates/fides_datamap_template.xlsx
+recursive-include src/ui-build *
```

Comment:
I suggest nesting this one deeper, as the `src` level isn't actually getting packaged (it is only packaging from `src/fidesctl/` and down I think)

Reply:
ahhh good point